lib/matplotlib/artist.py

```diff
@@ -1411,7 +1411,9 @@ def pprint_setters_rest(self, prop=None, leadingspace=4):
 
         attrs = sorted(self._get_setters_and_targets())
 
-        names = [self.aliased_name_rest(prop, target)
+        names = [self.aliased_name_rest(prop, target).replace(
```

Comment:
it doesn't have to be this PR, but perhaps we should just fix `Axes.__module__` to be `matplotlib.axes`, not `matplotlib.axes._base`?  see https://github.com/numpy/numpy/pull/12382 for similar stuff on numpy's side.

Reply:
That seems reasonable, but also push to another PR.